core/scripts/vrfv2/testnet/util.go

```diff
@@ -34,6 +38,20 @@ func deployBatchBHS(e helpers.Environment, bhsAddress common.Address) (batchBHSA
 	return helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID)
 }
 
+func deployLinkToken(e helpers.Environment) common.Address {
+	_, tx, _, err := link_token_interface.DeployLinkToken(e.Owner, e.Ec)
+	helpers.PanicErr(err)
+	return helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func deployLinkEthFeed(e helpers.Environment, linkAddress string, weiPerUnitLink decimal.Decimal) common.Address {
```

Comment:
Function signature nit: `weiPerUnitLink` is better served as `*big.Int`, the caller can just call `BigInt()` if they have a `decimal.Decimal` object.